packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -121,20 +133,27 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
                 </div>
               </div>
 
-              <div className={classNames.code}>{isCodeVisible && <CodeSnippet language="tsx">{code}</CodeSnippet>}</div>
-
-              {activeCustomizations ? (
-                <CustomizerContext.Provider value={{ customizations: { settings: {}, scopedSettings: {} } }}>
-                  <Customizer {...activeCustomizations}>
-                    <ThemeProvider scheme={_schemes[schemeIndex]}>
-                      <Stack styles={regionStyles}>{exampleCardContent}</Stack>
-                    </ThemeProvider>
-                  </Customizer>
-                </CustomizerContext.Provider>
-              ) : (
-                exampleCardContent
+              {isCodeVisible && !this.canRenderLiveEditor && (
+                <div className={classNames.code}>
+                  <CodeSnippet language="tsx">{code}</CodeSnippet>
+                </div>
               )}
 
+              {isCodeVisible && this.canRenderLiveEditor && editor}
```

Comment:
Instead of declaring the editor JSX up above (which results in always initializing the component even if it's not rendered), you should declare it inline here so it's only initialized when used.

Also, it would probably be better to do the conditional rendering like this:
```
{isCodeVisible && (this.canRenderLiveEditor ? <Editor .../> : /*div with CodeSnippet*/)}
```

Reply:
We also need to be sure that Monaco is delay-loaded before this goes in so it doesn't have too much perf impact.